ingest/ingest_files.py

```diff
@@ -227,9 +237,12 @@ def open_file(self, file_path, open_as=None, start_point: int = 0, **kwargs):
                     open_file,
                 )
         else:
-            raise ValueError(
-                f"Unsupported file format. Allowed file types are: {' '.join(self.allowed_file_types)}"
+            msg = (
+                f"Unsupported file format. Allowed file types are: "
```

Comment:
Revisiting this, I think users would usually consider "file type" to be synonymous with "file extension" -- which is not the case here.  Many users won't know what MIME types are off hand, but including "MIME" in the error message at least gives them a hook they could search, something slightly more actionable.

```suggestion
                f"Unsupported file format. Allowed file MIME types are: "
```

I don't feel strongly about it, just something for consideration.